### PR TITLE
Introduce Docker container setup

### PR DESCRIPTION
**Pull request by tokk-nv**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    jupyter lab build
+
+# install jetbot python module
+RUN cd /opt && \
+    git clone https://github.com/NVIDIA-AI-IOT/jetbot/ && \
+    cd jetbot && \
+    apt install -y python3-smbus && \
+    apt-get install -y cmake && \
+    python3 setup.py install && \
+    cp -r notebooks ${WORKDIR}/
```
Comment (jaybdub) [inline Dockerfile:60]: Is workdir in this instance the mounted volume?  Does this work?

Comment (tokk-nv) [reply Dockerfile:60]: no, so I took care of populating this with `docker_setup_workdir.sh`.
We cam remove this line.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+FROM ${BASE_IMAGE}
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV JUPYTER_PASSWORD=jetbot
+
+WORKDIR /jetbot_dir
+
+ENV SHELL /bin/bash
+
+RUN apt-get update && apt-get install -y
```
Comment (jaybdub) [inline Dockerfile:11]: Do you know if this line does anything?  Thinking of removing in my branch

Comment (tokk-nv) [reply Dockerfile:11]: We can remove this line.

**Pull request closed without merging**